### PR TITLE
WordPress Full Stack quickstart - creation

### PR DESCRIPTION
**Pull request by pkosanam**
Ticket ID: https://issues.newrelic.com/browse/NR-89158

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by pkosanam** [comment]
need to update the docs links, hence converting this PR to draft

File: quickstarts/wordpress-fullstack/config.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      Monitor the performance metrics of your WordPress application and its instances in real-time with New Relic.
+    url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/wordpress-fullstack-integration/
+keywords:
+  - wordpress
+  - php
+  - performance monitoring
+  - CMS
+  - wordpress full stack
+  - browser
+  - full stack
```
Comment (pkosanam) [inline quickstarts/wordpress-fullstack/config.yml:39]: add
```
  - word press
```

File: quickstarts/wordpress-fullstack/config.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  ### Why monitor WordPress?
+  Websites across various domains extensively utilize WordPress applications. Despite their versatility, these websites can experience slowdowns caused by factors such as slow-loading plugins, database connection problems, or other variables.
+  Our WordPress Full-Stack quickstart offers step-by-step documentation for instrumenting the quickstart and provides a pre-built dashboard to monitor crucial performance metrics, database metrics, and error messages generated on a specific URL. These indicators assist you and your team in identifying and addressing concerns, ultimately enhancing your website's performance.
+
+  ### What’s included in this quickstart?
+  New Relic WordPress Full Stack monitoring agent capture the following metrics:
+  - Easily identify the plugin(s) that consumes the most time and resources
+  - User usage statistics i.e., user device type and unique visitors
+  - Database metrics like connections per second and more
+  - Browser metrics like page view comparison, TTFB and more
```
[the remark this answers is not in the document]
Comment (pkosanam) [reply quickstarts/wordpress-fullstack/config.yml:17]: done

File: quickstarts/wordpress-fullstack/config.yml (hunk 1 above)
Comment (pkosanam) [reply quickstarts/wordpress-fullstack/config.yml:39]: done

**Review by seemantk — CHANGES_REQUESTED** [review]
minor language changes

File: quickstarts/wordpress-fullstack/config.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  Websites across various domains extensively utilize WordPress applications. Despite their versatility, these websites can experience slowdowns caused by factors such as slow-loading plugins, database connection problems, or other variables.
+  Our WordPress Full-Stack quickstart offers step-by-step documentation for instrumenting the quickstart and provides a pre-built dashboard to monitor crucial performance metrics, database metrics, and error messages generated on a specific URL. These indicators assist you and your team in identifying and addressing concerns, ultimately enhancing your website's performance.
+
+  ### What’s included in this quickstart?
+  New Relic WordPress Full Stack monitoring agent capture the following metrics:
+  - Easily identify the plugin(s) that consumes the most time and resources
+  - User usage statistics i.e., user device type and unique visitors
+  - Database metrics like connections per second and more
+  - Browser metrics like page view comparison, time to first byte (TTFB) and more
+  - Transactions and its error messages
```
[the remark this answers is not in the document]
Comment (pkosanam) [reply quickstarts/wordpress-fullstack/config.yml:18]: done

File: quickstarts/wordpress-fullstack/config.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+  New Relic WordPress Full Stack quickstart enables performance monitoring of your WordPress application through agents installation and New Relic Flex configuration.
+  
+  ### Why monitor WordPress?
+  Websites across various domains extensively utilize WordPress applications. Despite their versatility, these websites can experience slowdowns caused by factors such as slow-loading plugins, database connection problems, or other variables.
+  Our WordPress Full-Stack quickstart offers step-by-step documentation for instrumenting the quickstart and provides a pre-built dashboard to monitor crucial performance metrics, database metrics, and error messages generated on a specific URL. These indicators assist you and your team in identifying and addressing concerns, ultimately enhancing your website's performance.
+
+  ### What’s included in this quickstart?
+  New Relic WordPress Full Stack monitoring agent capture the following metrics:
+  - Easily identify the plugin(s) that consumes the most time and resources
```
[the remark this answers is not in the document]
Comment (seemantk) [reply quickstarts/wordpress-fullstack/config.yml]: @pkosanam the requested changes are not being seen, can you please verify?

**Review by seemantk — APPROVED** [review]
lgtm, thanks @pkosanam

**Pull request merged**